### PR TITLE
2.7 cleanup

### PR DESCRIPTION
**Pull request by FrancoisRegisDegott-eaton**
- cmake cleanup
- README updates
- remove logger.conf  (use ftylog.cfg default)
- complete 'fty-' prefix for bus addresses

### CODE REVIEW

File: packaging/debian/changelog [hunk 1]
```diff
@@ -2,4 +2,4 @@ fty-discovery-ng (1.0.0) stable; urgency=medium
 
   * Initial release.
 
- -- Clement Perrette <clementperrette@eaton.com>  Thu, 25 Apr 2020 09:42:40 +0100
+ -- fty-discovery-ng Developers <eatonproductfeedback@eaton.com>  Thu, 25 Apr 2020 09:42:40 +0100
```
Comment (aquette) [inline packaging/debian/changelog:5]: 👍

File: packaging/debian/control [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Source: fty-discovery-ng
-Maintainer: Clement Perrette <clementperrette@eaton.com>
+Maintainer: fty-discovery-ng Developers <eatonproductfeedback@eaton.com>
```
Comment (aquette) [inline packaging/debian/control:2]: 👍

File: rest/src/asset.cpp [hunk 3]
```diff
@@ -57,7 +57,7 @@ Expected<std::string> AssetRest::assets(const commands::assets::In& param)
     disco::Message msg;
     msg.userData.setString(*pack::json::serialize(param));
 
-    msg.meta.to      = "discovery-ng";
+    msg.meta.to      = "fty-discovery-ng";
```
Comment (aquette) [inline rest/src/asset.cpp:60]: Beware that it's still registering as disco-ng, not fty

File: rest/src/discovery-rest.h [hunk 4]
```diff
@@ -9,11 +9,11 @@ static constexpr const char* AgentName = "discovery-ng_rest";
 inline fty::disco::Message message(const std::string& subj)
 {
     fty::disco::Message msg;
-    msg.meta.to      = "discovery-ng";
+    msg.meta.to      = "fty-discovery-ng";
```
Comment (aquette) [inline rest/src/discovery-rest.h:12]: are you 100% sure that all other agents / rest will query the right name, and not just disco-ng?

**Review by aquette — CHANGES_REQUESTED** [review]

File: rest/src/asset.cpp (hunk 3 above)
Comment (FrancoisRegisDegott-eaton) [reply rest/src/asset.cpp:60]: fty- prefix is applied everywhere

File: rest/src/discovery-rest.h (hunk 4 above)
Comment (FrancoisRegisDegott-eaton) [reply rest/src/discovery-rest.h:12]: Yes, only the RESt client request the fty-discovery-ng service
(I've made a global search on all fty- etn- repos...)

**Review by aquette — APPROVED** [review]

**Comment by aquette** [comment]
post D3 please @jeffdegott

**Review by ericclappier-eaton — APPROVED** [review]

**Pull request merged**